query-engine/connectors/sql-query-connector/src/row.rs

```diff
@@ -171,3 +189,84 @@ impl From<&SqlId> for DatabaseValue<'static> {
         id.clone().into()
     }
 }
+
+// We assume the bytes are stored as a big endian signed integer, because that is what
+// mysql does if you enter a numeric value for a bits column.
+fn interpret_bytes_as_i64(bytes: &[u8]) -> i64 {
```

Comment:
So I'm not proposing or asking any changes, just writing down here that I'm a bit uncomfortable with this :)

Reply:
Yes, it would be nice to have a way to specialize things depending on the specific database we're working with in the sql query connector. That way we could do this, but only on mysql. This hack may go away when we expand the type system.